src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/InputPackageAnnotation2.java

```diff
@@ -9,6 +9,6 @@
 @Deprecated
 package com.puppycrawl.tools.checkstyle.checks.annotation.packageannotation;
 
-public class InputPackageAnnotation2 {
+public class InputPackageAnnotation2 { // ok
```

Comment:
could you please explain why `// ok` comments were added?

Reply:
It was a slight mistake, most of them are actually files with no violations. However, a couple of them were mistaken.
romani said we don't care about violation comments right now, we'll place them during migration.

Reply:
I've fixed these, however